install/overlays/vsphere/kustomization.yaml

```diff
@@ -18,9 +18,32 @@ configMapGenerator:
   namespace: confidential-containers-system
   literals:
   - CLOUD_PROVIDER="vsphere"
-  - GOVC_URL="" # set
+  - GOVC_URL="" # set of the vCenter URL is required.
+  - GOVC_DATACENTER="" # set of the vCenter datacenter is required.
+
+  #- GOVC_TEMPLATE=""  # Uncomment and set if you want to clone the peerpod VM from an existing
+                       # GOVC_DATACENTER template other than the default name podvm-template.
+
+  #- GOVC_DRS=""       # Uncomment and set to true if you want vCenter DRS to determine the peerpod VM placement.
+                       # A DRS Automation=Manual configured GOVC_VCLUSTER in your GOVC_DATACENTER must be indicated as well.
+
+  #- GOVC_VCLUSTER=""  # Uncomment and set to place your peerpod VM on an existing vCenter cluster in your GOVC_DATACENTER.
+                       # If GOVC_DRS is selected a DRS cluster is required.
+
+  #- GOVC_HOST=""      # Uncomment and set to place your peerpod VM on an existing host in your GOVC_DATACENTER.
+                       # If not using DRS a host name is required.
+                       # If using DRS the host name is ignored.
+
+  #- GOVC_DATASTORE="" # Uncomment and set to place your peerpod VM on your GOVC_HOST datastore in your GOVC_DATACENTER.
+                       # If not using DRS a datastore name is required.
+                       # If using DRS the datastore name is ignored.
+
+  #- GOVC_FOLDER=""     # Uncomment to place your peerpod VM in an existing desintation folder
+                        # or create a new one if it does not exist in the VM inventory path
+                        # (GOVC_DATACENTER/vm/GOVC_FOLDER).
+
   #- PAUSE_IMAGE="" # Uncomment and set if you want to use a specific pause image
-  #- VXLAN_PORT="" # Uncomment and set if you want to use a specific vxlan port. Defaults to 4789
+  - VXLAN_PORT="" # set to use "9000" or change if you want to use a specific vxlan port. Defaults to 4789.
```

Comment:
better to keep commented if not mandatory

Reply:
K I'll fix this.